### ISSUE TITLE
Ignoring a namespace for jobs ignores it also for deployments

### ISSUE THREAD

**Issue by juissi-t** [kubeshop/botkube#669]
## Description

We're using Botkube version 0.12.4. Our config file includes resources for deployments and jobs, similar but with one exception: a couple more namespaces are ignored for jobs than for deployments. Whenever a deployment is updated in the namespace where changes to jobs are ignored, we get no notification for the changes done. But if we remove the namespace from being ignored, we do get notifications for both jobs and deployments.

## Expected behavior

When ignoring a namespace for jobs, we expect to still get notifications for changes to deployments.

## Actual behavior

When ignoring jobs in a namespace, it looks like Botkube also ignores changes to deployments.

## Steps to reproduce

We deploy Botkube using Flux with the following config, which shows the Helm chart values:

```
---
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  name: botkube
  namespace: botkube
spec:
  chart:
    spec:
      chart: botkube
      sourceRef:
        kind: HelmRepository
        name: infracloudio
      version: v0.12.4
  install: {}
  interval: 10m0s
  values:
    config:
      resources:
        - name: apps/v1/deployments
          namespaces:
            include:
              - all
            ignore:
              - amazon-cloudwatch
              - botkube
              - external-secrets
              - flux-system
              - kube-system
              - kyverno
              - monitoring
          events:
            - create
            - update
            - delete
            - error
          updateSetting:
            includeDiff: true
            fields:
              - spec.template.spec.containers[*].image
        - name: batch/v1/jobs
          namespaces:
            include:
              - all
            ignore:
              - amazon-cloudwatch
              - botkube
              - external-secrets
              - flux-system
              - kube-system
              - kyverno
              - monitoring
              - ingress-nginx
              - api-repo-workers
          events:
            - create
            - update
            - delete
            - error
          updateSetting:
            includeDiff: true
            fields:
              - spec.template.spec.containers[*].image
              - status.conditions[*].type
    communications:
      existingSecretName: botkube-communications
    serviceMonitor:
      enabled: true
      interval: 30s
    resources:
      limits:
        cpu: 200m
        memory: 128Mi
      requests:
        cpu: 100m
        memory: 128Mi
```

If we remove `api-repo-workers` from the list of ignored namespaces for jobs and re-deploy, we get notifications for changes in deployments in the namespace.

**Comment by ezodude** [comment]
@juissi-t yup, I can concur. I can also replicate this issue on Botkube version 0.12.4.

But, I also tested against the [latest dev build](https://github.com/kubeshop/botkube/pkgs/container/botkube/30881190?tag=v9.99.9-dev) and it appears this bug has been resolved.

The caveat here is: the latest dev build uses an [updated version of the config](https://github.com/kubeshop/botkube/blob/main/helm/botkube/values.yaml).

You'll need to update your YAML config to be able to run this version of BotKube.

**Comment by pkosiec** [comment]
Closing as resolved then 🙂 Final BotKube 0.13 version is just around the corner and you can try out [0.13 RC](https://github.com/kubeshop/botkube/releases) 🙂

**Comment by juissi-t** [comment]
Thanks, I'll try out 0.13 next week. :smile:
